packages/skeleton-svelte/src/components/Tabs/TabsControl.svelte

```diff
@@ -3,6 +3,9 @@
 	import type { TabsControlProps } from './types.js';
 
 	const {
+		// New prop to choose element type
+		as = 'button', // 'button' or 'a'
```

Comment:
No need for an explicit prop as this should be determined if an `href` is present. Zag likely provides this as part of the `ctx` or `state`. From their docs looks like there should be a [navigate() method](https://zagjs.com/components/react/tabs#machine-context) for this purpose.